books/free-programming-books-subjects.md

```diff
@@ -102,6 +102,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Elementary Algorithms](https://github.com/liuxinyu95/AlgoXY) - Larry LIU Xinyu
 * [Foundations of Computer Science](http://infolab.stanford.edu/~ullman/focs.html) - Al Aho and Jeff Ullman
 * [Geometry Algorithms](http://geomalgorithms.com) - Dan Sunday
+* [Introduction to Algorithms, Third Edition](https://web.iiit.ac.in/~pratik.kamble/storage/Algorithms/Cormen_Algorithms_3rd.pdf) - Thomas H. Cormen, Charles E. Leiserson, Ronald L. Rivest, and Clifford Stein (PDF)
```

Comment:
:information_source: I think this is the right site: https://mitpress.mit.edu/books/introduction-algorithms-third-edition

:warning: Not free :question:. Neither of authors is Pratik Kamble. That a link is public on the internet does not mean that its content is free. See [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#guidelines)

:information_source: Also edition / year goes between parenthesis for ancient versions.

Reply:
Ok, I will take note of this 👍  😄